fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -112,14 +113,14 @@ sealed interface ReviewQualityCheckState : State {
                 val reviewGrade: Grade?,
                 val analysisStatus: AnalysisStatus,
                 val adjustedRating: Float?,
+                val notEnoughReviews: Boolean,
                 val productUrl: String,
                 val highlightsInfo: HighlightsInfo?,
                 val recommendedProductState: RecommendedProductState = RecommendedProductState.Initial,
             ) : ProductReviewState {
                 init {
-                    require(!(highlightsInfo == null && reviewGrade == null && adjustedRating == null)) {
-                        "AnalysisPresent state should only be created when at least one of " +
-                            "reviewGrade, adjustedRating or highlights is not null"
+                    require(!notEnoughReviews) {
+                        "AnalysisPresent state should only be created if there are enough reviews"
```

Comment:
I don't think we need to add this boolean to `AnalysisPresent` since `Error.NotEnoughReviews` is an error UI state and is already covered.

Also, I think we should keep the existing require condition, since that's still valid.